msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md

```diff
@@ -8,6 +8,11 @@ ms.date: 02/27/2023
 
 # Create Outgoing Webhooks
 
+> [!IMPORTANT]
```

Comment:
Outgoing webhook is not affected. Please remove deprecation message from Outgoing Webhooks.

Reply:
Updated.